q2_phylogeny/plugin_setup.py

```diff
@@ -60,6 +62,33 @@
     citations=[citations['price2010fasttree']]
 )
 
+plugin.methods.register_function(
+    function=q2_phylogeny.raxml,
+    inputs={
+            'alignment': FeatureData[AlignedSequence]},
+    parameters={
+            'seed': Int,
+            'n_threads': Int % Range(1, None),
+            'substitution_model': Str % Choices(_RAXML_MODEL_OPT)},
+    outputs=[('tree', Phylogeny[Unrooted])],
+    input_descriptions={
+        'alignment': ('Aligned sequences to be used for phylogenetic '
+                      'reconstruction.'),
+    },
+    parameter_descriptions={
+        'n_threads': ('The number of threads. Using more than one thread '
+                      'will enable the PTHREADS version of RAxML'),
+        'substitution_model': ('Model of Nucleotide Substitution'),
+        'seed': ('Random number seed for the parsimony starting tree.'
```

Comment:
Missing a space at the end of the line (`starting tree. `). These ones are super tricky, and are normally only caught when looking at the rendered help text via the cli interface (or the Artifact API docstring).